ci/actions-templates/macos-builds-template.yaml

```diff
@@ -4,8 +4,14 @@ jobs: # skip-x86_64 skip-aarch64
   # Do not edit this file in .github/workflows
   build-macos-aarch64: # job-name skip-x86_64
     runs-on: macos-14 # skip-x86_64
+    env: # skip-x86_64
+      MACOSX_DEPLOYMENT_TARGET: 11.0 # skip-x86_64
+      MACOSX_STD_DEPLOYMENT_TARGET: 11.0 # skip-x86_64
   build-macos-x86_64: # job-name skip-aarch64
     runs-on: macos-13 # skip-aarch64
+    env: # skip-aarch64
+      MACOSX_DEPLOYMENT_TARGET: 10.12 # skip-aarch64
+      MACOSX_STD_DEPLOYMENT_TARGET: 10.12 # skip-aarch64
```

Comment:
`MACOSX_STD_DEPLOYMENT_TARGET` should not be set. That is something bootstrap uses when building the standard library.

Reply:
@ehuss Thanks! I've fixed the code accordingly.